config/crd/bases/lvm.openshift.io_topolvmclusters.yaml

```diff
@@ -0,0 +1,217 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: topolvmclusters.lvm.openshift.io
```

Comment:
I think we should avoid explicit references to openshift (or any other RH product) . IMHO one of main targets should be to produce an operator which will be easily included under the topolvm umbrella.

Reply:
Sounds good! Looks like topolvm-operator [might be planning to switch to topolvm.io](https://github.com/topolvm/topolvm/issues/168)